src/WMOToast/WMOToastRedux.js

```diff
@@ -0,0 +1,103 @@
+import Immutable from 'seamless-immutable';
+
+import { stringArrayToObject } from '../redux/reduxUtils';
+
+/* ------------- Toast actions ------------- */
+
+export const actions = stringArrayToObject(['DISPLAY_ERROR', 'DISPLAY_WARNING', 'DISPLAY_INFO', 'HIDE']);
+
+export const actionCreators = {
+  hide() {
+    return {
+      type: actions.HIDE
+    };
+  },
+  displayError(message, duration = 4000) {
+    return {
+      type: actions.DISPLAY_ERROR,
+      payload: {
+        message,
+        duration
+      }
+    };
+  },
+  displayWarning(message, duration = 4000) {
+    return {
+      type: actions.DISPLAY_WARNING,
+      payload: {
+        message,
+        duration
+      }
+    };
+  },
+  displayInfo(message, duration = 4000) {
+    return {
+      type: actions.DISPLAY_INFO,
+      payload: {
+        message,
+        duration
+      }
+    };
+  }
+};
+
+/* ------------- Toast reducer ------------- */
+
+const defaultState = {
+  message: null,
+  error: false,
+  warning: false,
```

Comment:
`warning` and `error` are pretty standard words for logging and showing alerts. I'd keep them.
To customize the toast you can use the style props that the container have.
What do you think?